R/pagination.R

```diff
@@ -958,6 +958,8 @@ paginate_to_mpfs <- function(obj,
                       nosplitin = nosplitin,
                       verbose = verbose)
         return(unlist(deep_pag, recursive = FALSE))
+    } else if (!is.null(attr(obj, "page_title_prefix")) && !is.na(attr(obj, "page_title_prefix"))) {
```

Comment:
@gmbecker in `rtables` there are functions `ptitle_prefix` and `has_force_pag` which are used to do these checks (and are used in `paginate_table`). Should these functions be moved to `formatters` so that they can be used here as well?

Reply:
so prtitle_prefix is what the parent carries around that gets inserted into the page titles, but we should be able to just check `page_title(fpags[[1]])` to see if stuff is in there. 

Please confirm this with tests of both cases @edelarua

Reply:
To explain further, `has_force_pag` is to detect whether forced pagination is needed. So we could refactor how `paginate_to_mpfs` works to check explicitly, but as it is now we are trying to determine whether it happened *after* it occurs. 

We should probably wrap the check I suggested above in something like `has_page_title`, or something